GliaWidgets/Coordinator/RootCoordinator.swift

```diff
@@ -136,15 +135,22 @@ class RootCoordinator: SubFlowCoordinator, FlowCoordinator {
             case .back:
                 switch self?.engagement {
                 case .chat:
-                    self?.gliaViewController?.minimize(animated: true)
+                    if !(self?.interactor.isEngaged ?? false) {
+                        self?.popCoordinator()
+                        self?.end()
+                    } else {
+                        self?.gliaViewController?.minimize(animated: true)
+                    }
```

Comment:
imho, it is easier to read code when it doesn't contain additional negation `!`. So, I'd like to rework it a little bit
```swift
if self?.interactor.isEngaged == true {
    self?.gliaViewController?.minimize(animated: true)
} else {
    self?.popCoordinator()
    self?.end()
}
```

Reply:
Good catch. Had to refactor it anyway for bubble logic, should be good now